consensus/consensus_v2.go

```diff
@@ -617,7 +617,7 @@ func (consensus *Consensus) finalizeCommits() {
 	// In current code, we add signatures in block in tryCatchup, the block dump to explorer does not contains signatures
 	// but since explorer doesn't need signatures, it should be fine
 	// in future, we will move signatures to next block
-	explorer.GetStorageInstance(consensus.leader.IP, consensus.leader.Port, true).Dump(block, beforeCatchupViewID)
+	explorer.GetStorageInstance(consensus.leader.IP, consensus.leader.Port, true).Dump(block, beforeCatchupNum)
```

Comment:
why change this one?

Reply:
This was wrong. It should be block number not view ID for identifying blocks.